python/flow_sdk/shim_singer_sdk.py

```diff
@@ -240,12 +251,7 @@ def sync_streams(self, delegate: singer_sdk.Tap):
     "type": "object",
     "properties": {
         "stream": {"type": "string"},
-        "syncMode": {"type": "string", "enum": ["incremental", "full_refresh"]},
-        "namespace": {"type": "string"},
-        "cursorField": {
-            "type": "array",
-            "items": {"type": "string"},
-        },
+        "replication_mode": {"type": "string", "enum": ["INCREMENTAL", "FULL_TABLE"]},
```

Comment:
i see `replication_method` above, but `replication_mode` here. Also, should `replication_key` be schematized ?

Reply:
Wow, that's a typo. Both done.